src/m365/spo/commands/listitem/listitem-roleinheritance-break.ts

```diff
@@ -0,0 +1,107 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { validation } from '../../../../utils';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  id: number;
+  listId?: string;
+  listTitle?: string;
+  clearExistingPermissions?: boolean;
+}
+
+class SpoListItemRoleInHeritanceBreakCommand extends SpoCommand {
```

Comment:
Class name should be `SpoListItemRoleInheritanceBreakCommand`

Reply:
done